src/js/modules/breakpointHelper.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @overview Breakpoint handler
+ *
+ * @module breakpointHelper
+ */
+
+export const getBreakpoints = () => {
+    const output = {};
+
+    // Append hidden element to body
+    const screenSizer = document.createElement('div');
+    screenSizer.classList.add('c-screen-sizer');
+
+    document.body.appendChild(screenSizer);
+
+    // It should have a 'content' property containing the breakpoints
+    const breakpoints = window.getComputedStyle(document.querySelector('.c-screen-sizer'))
+        .getPropertyValue('content')
+        .replace(/"/g, '')
```

Comment:
Do you need these two lines? Would there ever be any quotes in the value? If you do need them you should be able to reduce to a single line: `.replace(/['"]+/g, '')`

Reply:
They are needed, as the value returned is a bit of a weird format – might be able to reduce it down though